plugins/dev-tools/src/playground/index.js

```diff
@@ -191,6 +193,7 @@ export function createPlayground(
           currentTab = tab;
           currentGenerate = tab.generate;
           const isXml = tab.state.name == 'XML';
+          const isJson = tab.state.name == 'JSON';
```

Comment:
Since `isXml` and `isJson` are mutually exclusive (but not exhaustive), might it be better for the two booleans to be replaced by an enum?

Reply:
I think they need to be booleans to set the context keys below.

If they weren't just local to this function I would definitely change it, but since they're only used in this one spot I think keeping them as booleans makes more sense.